web/src/components/admin/settings/queue/AdminQueueStats.vue

```diff
@@ -15,10 +15,10 @@
               {{ stats.completed_count }}
             </h4>
           </div>
-          <div class="pb-4 lg:pb-6">
+          <div v-if="total > 0" class="pb-4 lg:pb-6">
```

Comment:
Total IMO is the capacity your system with all agents can handle. So if there are 0 tasks it would be 100% green / free. If you have no agents and no tasks it should probably have some backgroundish color.